sandbox/docker/master/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
+
+WORKDIR /tmp
+COPY ./requirements.txt /tmp/
+
+RUN apt-get update; \
+        && apt-get install -y git; \
+        && pip install -r requirements.txt \
+        && rm -rf /tmp/requirements.txt \
+        && rm -rf /root/.cache
```

Comment:
Can we run `apt-get purge -y git` and `apt-get autoremove -y` in the end to reduce the size of Docker image?

Reply:
Yeah, or a multi stage build, good idea

Reply:
I changed it to a multistage build.

If you think that's over complicating it I'll just change it to include the git purge and autoremove.

I think the final image will be smaller this way though, because of apt nonsense.

Reply:
I think we need to copy files other than `site-packages` as well to make a multi stage build approach work. Just removing unnecessary packages with apt may be easier for our use case.

Reply:
`apt` is leaving behind a bunch of log files and stuff, I think all needed files are correctly copied now.